.prettierignore

```diff
@@ -1,2 +1,3 @@
 # Avoid yarn and prettier fighting over formatting
 package.json
+**/*.trace.json
```

Comment:
What's this?

Reply:
We have sample traces, basically JSON files that our benchmarks and tests use, there's not need to prettify them

Reply:
Sounds pretty unrelated to this TSC performance fix, should we put it in a diff PR?

Reply:
I mean, sure, but I think it's so small that we dont have to care tbh